start-kafka-lite.sh

```diff
@@ -1,16 +1,36 @@
 #!/bin/bash
 
+# Create Kafka properties file
+echo "enable SASL and  PLAINTEXT authentication"
 cat > ./kafka.properties <<EOL
 authorizer.class.name=kafka.security.authorizer.AclAuthorizer
 broker.id=1
 cluster.id=$KAFKA_CLUSTER_ID
-listeners=PLAINTEXT://:$KAFKA_PORT
 zookeeper.connect=localhost:$ZOOKEEPER_PORT
 log.dirs=$KAFKA_DATA_DIR
 offsets.topic.replication.factor=1
 transaction.state.log.replication.factor=1
 transaction.state.log.min.isr=1
+listeners=SASL_PLAINTEXT://:$KAFKA_PORT,PLAINTEXT://:$KAFKA_NOAUTH_PORT
+super.users=User:admin
+sasl.enabled.mechanisms=PLAIN
+security.inter.broker.protocol=SASL_PLAINTEXT
+sasl.mechanism.inter.broker.protocol=PLAIN
+
+EOL
+
+# Create JAAS configuration file for Kafka
+cat > ./kafka_jaas.conf <<EOL
+KafkaServer {
+  org.apache.kafka.common.security.plain.PlainLoginModule required
+  username="admin"
+  password="admin-secret"
+  user_admin="admin-secret";
```

Comment:
I'd probably go with `admin:admin` as username:password to match what we do with Postgres.

Also, would it be worth adding an additional non-admin user for (potentially) more realistic integration testing?  I can't find concrete documentation here, but it seems like that might just be an extra line here:

```suggestion
  password="admin"
  user_admin="admin"
  user_user="user";
```